include/ocpp/v201/ocpp_types.hpp

```diff
@@ -890,7 +1992,7 @@ void from_json(const json& j, SetMonitoringResult& k);
 std::ostream& operator<<(std::ostream& os, const SetMonitoringResult& k);
 
 struct SetVariableData {
-    CiString<1000> attributeValue;
+    CiString<2500> attributeValue;
```

Comment:
TODO: find out implications for backwards compatibility with 2.0.1

Reply:
Accept